schema/trivia.schema.json

```
@@ -0,0 +1,52 @@
+{
+  "$id": "https://raw.githubusercontent.com/Cog-Creators/Red-DiscordBot/V3/develop/schema/trivia.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Red-DiscordBot Trivia List file",
+  "type": "object",
+  "properties": {
```

I'm unsure this is required but some schema validators seem to reject `$schema` field as having an invalid type when `additionalProperties` is set to an array of strings so it's probably better to define it:
```suggestion
  "properties": {
    "$schema": {
      "type": "string",
      "format": "uri"
    },
```